resources/views/components/alert.blade.php

```diff
@@ -0,0 +1,10 @@
+<div class="my-4 px-3 py-2 border rounded-sm
+  @switch($type ?? null)
+    @case('success') bg-green-200 text-green-800 border-green-500 @break
+    @case('warning') bg-yellow-200 text-yellow-800 border-yellow-500 @break
+    @case('error') bg-red-200 text-red-700 border-red-500 @break
+    @default bg-indigo-200 text-indigo-800 border-indigo-500
+  @endswitch
```

Comment:
This whole bit may be particularly revolting or just a bit odd for those unfamiliar with this method of writing CSS/building components, however, I could not bring myself to remove the class names from the view and chuck them in a View Composer (would require Purgecss rules to be adjusted) or extract them a component as, IMO that defeats the purpose Tailwind aims to solve.

This is the one example component that was left in Sage and I think it does good job cover how to create aliased Blade/Tailwind components with multiple states since it uses a service provider and components (which saw little use in Sage 9).

I’d like to hear what other think though.